README.md

```diff
@@ -22,32 +22,41 @@
 [![License](http://img.shields.io/:license-Apache%202-blue.svg)](https://github.com/apache/arrow-adbc/blob/master/LICENSE.txt)
 
 ADBC is an API standard (version 1.0.0) for database access libraries ("drivers") in C, Go, and Java that uses Arrow for result sets and query parameters.
-Instead of writing code for each individual database, applications can build against the ADBC APIs, and link against drivers that implement the standard.
+Instead of writing code to extract Arrow data out of each individual database, applications can build against the ADBC APIs, and link against drivers that implement the standard.
 Additionally, a JDBC/ODBC-style driver manager is provided. This also implements the ADBC APIs, but dynamically loads drivers and dispatches calls to them.
 
-Like JDBC/ODBC, the goal is to provide a generic API for multiple databases, but ADBC is focused on Arrow-based data access for analytics use cases (bulk data retrieval/ingestion), and not the full spectrum of use cases that JDBC/ODBC drivers handle.
+Like JDBC/ODBC, the goal is to provide a generic API for multiple databases.
+ADBC, however, focuses on Arrow-based data access for analytics use cases - primarily bulk data retrieval and ingestion, and not the full spectrum of use cases that JDBC/ODBC support.
```

Comment:
Maybe this?

```suggestion
Like JDBC/ODBC, the goal is to provide a generic API for multiple databases. ADBC, however, is focused on bulk columnar data retrieval and ingestion through an Arrow-based API rather than attempting to replace JDBC/ODBC in all use cases.
```